visual-tests/tests/.empty-test-template.js

```diff
@@ -0,0 +1,29 @@
+import { helpers } from '../imports/helpers';
+
+const { test, expect } = require('@playwright/test');
+
+/* ======= */
```

Comment:
I did it because these fiiles will be used by less technical people also and wanted to make things easier for them, like "do not worry, you do not have to even look here". I removed first separator (line 5), what do you think about second one, which marks "end of configuration, beginning of test" (line 23 in all of files)?

Reply:
I’d remove all these comments

Reply:
done - removed from all of files.